workspaces/ui-v2/src/optic-components/hooks/config/AppConfiguration.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react';
+
+interface IFeatureFlags {}
+
+interface IAppConfigurations {
+  navigation: {
+    showDocs?: boolean;
+    showChangelog?: boolean;
+    showDiff?: boolean;
```

Comment:
nit: It might be nice to make these non-optional (removing the `?`) since we can cooerce undefined -> false. Makes downstream checks only need to look for false / undefined (usually not an issue though)

Reply:
agreed